tslint.json

```diff
@@ -1,5 +1,9 @@
 {
   "extends": [
     "./node_modules/@ht2-labs/typescript-project/configs/tslint.json"
-  ]
+  ],
+  "rules": {
+    "no-unused": false,
+    "no-use-before-declare": false
```

Comment:
@paul-stoner , what was the reason you changed this? 🤔

Reply:
@crazy-grizzly hey mate! 👋 

I added `no-unused` as for `jscommons/src/expressPresenter/handleCustomRoute.ts` we are getting the following error:
![image](https://user-images.githubusercontent.com/44973285/175925821-e6da3d9e-247e-491e-9ac8-1a34b680580f.png)

This error also occurs even after adding `/* tslint:disable no-unused no-magic-numbers */` to the opening line of scommons/src/expressPresenter/handleCustomRoute.ts. I think it's because it's added to our shared tsconfig.json file in the LP typescript-project repo [here](https://github.com/HT2-Labs/typescript-project/blob/master/configs/tslint.json#L7).

I added `no-use-before-declare` because it's no added to the shared [LP typescript-project](https://github.com/HT2-Labs/typescript-project/blob/master/configs/tslint.json) repo (can add here instead if needed?) and without we get the following depreciated error:
![image](https://user-images.githubusercontent.com/44973285/175926456-1fb13b43-c6b5-4149-ae47-4352dbc091b3.png)

Is this OK to merge still or would you want me to make any further changes? :)

Reply:
@paul-stoner , that is fine, I also added a change 👀